dkg/disk.go

```diff
@@ -59,6 +60,18 @@ func loadDefinition(ctx context.Context, conf Config) (cluster.Definition, error
 		return cluster.Definition{}, errors.Wrap(err, "unmarshal definition")
 	}
 
+	// Verify config hash and definition hash from json string and resultant cluster.Definition.
+	if !conf.NoVerify {
+		expected, err := json.MarshalIndent(res, "", " ")
```

Comment:
requiring a fixed json indentation as part of verification isn't correct. The json indentation shouldn't matter.

Reply:
yeah right, corrected 👍